src/components/UI/Card/CartCards/CartCards.jsx

```diff
@@ -24,18 +26,39 @@ function CartCards(props) {
             console.log("error ",err.message);
         }
     }
+
+    const AddProductsInWishListHandler = async (item)=>{
```

Comment:
Same here. Should be **addProductsInWishListHandler**. Try to make variable and function names smaller but descriptive. 
Ex:
AddProductsInWishListHandler can be **addToWishlistHandler**

Reply:
okay thanks for the useful comment i will make the changes